datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/types/common/mappers/CostMapper.java

```diff
@@ -0,0 +1,24 @@
+package com.linkedin.datahub.graphql.types.common.mappers;
+
+import com.linkedin.datahub.graphql.generated.Cost;
+import com.linkedin.datahub.graphql.generated.CostType;
+import com.linkedin.datahub.graphql.types.mappers.ModelMapper;
+
+import lombok.NonNull;
+
+public class CostMapper implements ModelMapper<com.linkedin.common.Cost, Cost> {
+
+    public static final CostMapper INSTANCE = new CostMapper();
+
+    public static Cost map(@NonNull com.linkedin.common.Cost cost) {
+        return INSTANCE.apply(cost);
+    }
+
+    @Override
+    public Cost apply(com.linkedin.common.Cost cost) {
```

Comment:
Is cost common? If not, we can create a new ML models-specific directory to contain these mappers

Reply:
Yes, `Cost` is part of common. Internally we added Cost as an Aspect to Dataset too.